### PR TITLE
Move to 2.60.2 LTS

### PR DESCRIPTION
**Pull request by jtescher**
Notable changes at: https://jenkins.io/changelog-stable/#v2.60.2

### CODE REVIEW

File: 2/Dockerfile [hunk 1]
```diff
@@ -22,7 +22,7 @@ ENV JENKINS_VERSION=2.60.1 \
     LC_ALL=en_US.UTF-8
 
 LABEL k8s.io.description="Jenkins is a continuous integration server" \
-      k8s.io.display-name="Jenkins 2.60.1" \
+      k8s.io.display-name="Jenkins 2.60.2" \
```
Comment (bparees) [inline 2/Dockerfile]: can you update the env on line 17 also?  thanks.

**Comment by bparees** [comment]
[test]

File: 2/Dockerfile (hunk 1 above)
Comment (jtescher) [inline 2/Dockerfile]: Good catch, not sure how that got missed. Fixed!

**Comment by bparees** [comment]
thanks!
[merge]

**Comment by bparees** [comment]
[merge]

**Comment by bparees** [comment]
[test]

**Comment by bparees** [comment]
[merge]

**Pull request merged**